cartridges/int_algolia/cartridge/scripts/algolia/lib/contentUtil.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+var URLUtils = require('dw/web/URLUtils');
+var jobHelper = require('*/cartridge/scripts/algolia/helper/jobHelper');
+var logger = jobHelper.getAlgoliaLogger();
+
+var contentAssetFunctions = {
+    // Generates a static URL based on the relative path provided
+    staticURL: function(relPath) {
+        return URLUtils.httpsStatic(relPath);
+    },
+    // A higher-order function to reduce duplication in creating URL functions
+    createUrlFunction: function(method) {
+        return function() {
+            var args = Array.prototype.slice.call(arguments);
+            var action = args.shift();
+            try {
+                return URLUtils[method].apply(URLUtils, [action].concat(args));
```

Comment:
Which extracting `action` to concat it back? Isn't it equivalent directly using `args`?
```suggestion
                return URLUtils[method].apply(URLUtils, args);
```

Reply:
Alright, removing the staticlink handling fixed my problem :+1: 
I think you've missed this comment, the rest looks good otherwise.